ai_council/execution/worker.py

```
@@ -0,0 +1,174 @@
+        """Start the worker node listening loop."""
+        await self._ensure_connection()
+        logger.info(f"Worker node starting. Listening on {self.task_queue}...")
+        
+        try:
+            while True:
+                # BLPOP from the queue (blocks until a message arrives)
+                result = await self.redis_client.blpop(self.task_queue, timeout=0)
+                if result:
+                    _, payload = result
+                    # Spawn a concurrent task so worker isn't blocked by one request
+                    asyncio.create_task(self.process_task(payload))
```

Ruff (0.15.7)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)